tests/h/services/delete_user_test.py

```diff
@@ -20,7 +21,7 @@ def test_delete_disassociate_group_memberships(self, factories, svc):
         assert user.groups == []
 
     def test_delete_deletes_annotations(
```

Comment:
I'd like to change the names of these tests but I figured there were enough changes in this PR already so I'll do that in the follow up when I convert the other db calls to use the corresponding service. I'd also like to change the names of the rest of the delete services to name_delete instead but again I think there are already so many changes in this PR.